include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -220,7 +220,11 @@ class FEEvaluationBase
    * sums them into the vector @p dst. The function also applies constraints
    * during the write operation. The functionality is hence similar to the
    * function AffineConstraints::distribute_local_to_global. If vectorization
-   * is enabled, the DoF values for several cells are used.
+   * is enabled, the DoF values for several cells are used. The mask can be
+   * used to suppress the write access for some of the vectorized cells, e.g.
+   * in case of local time stepping, where some cells are excluded from a
+   * call. A value of `true` in the bitset means that the respective lane
+   * index will be processed, whereas a value of `false` skips this index.
```

Comment:
Can you add: The default setting is a bitset that contains all ones, which will write the accumulated integrals to all cells in the batch.